### DISCUSSION TITLE
blog/computers-write-poetry

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#52] [Announcements]
# blog/computers-write-poetry

可能，刘慈欣的整个讨论充满了对数学、心灵、观念、存在等问题的基本误解，以十分粗糙的还原论来讲述了这么一个在哲学和科学上不可能的故事。

https://pathos.page/blog/computers-write-poetry

**Comment by astro-jingtao** [comment]
很有意思的博客。

感觉刘慈欣所谓的迭代器就是遍历 $m$ 个位置（取决于体裁），每个位置考虑全部的汉字（设为 $N$ 个），就生成 $N^m$ 种可能的组合。。。也许会根据格律要求加些限制（以小说中给的例子是没有的）。

后面又讲，能生成出来但没办法找出来所谓优秀作品，我认为实际上的问题就是博主说的”唐诗在首要意义上是一堆有物理形式（书写和声音）的对象 $x$，中国人将其把握为唐诗 $t$，神级文明将其把握为伪唐诗 $t'$“

当然他们可以找一大堆中国人（其可以从 $x$ 把握到 $t = h(x)$，忽略不同中国人的 $h()$ 也不同）给唐诗评分（假设这是 $s = g(t)$，且假设这个评分行为是有意义的），训练一个输入为 $x$ 输出为评分 $s$ 的神经网路或者什么别的更厉害的东西 $f()$ 使得 $f(x) = g(h(x))$. 但其实还是改变不了 $t'$ 不是 $t$ 的问题。
